python/srv.py

```diff
@@ -25,23 +25,34 @@
 
 logging.basicConfig(level=logging.INFO)
 
+parser = argparse.ArgumentParser()
+parser.add_argument(
+    'database', help='path to database file, in-memory will be used if empty')
+parser.add_argument(
+    'url', help='either UNIX:<socket_path> or <ip>:<tcp port> to listen on')
```

Comment:
Change part of the help to: `[<ip>:]<tcp port>` as this is current (desired?) behaviour